indra/assemblers/html/assembler.py

```diff
@@ -106,6 +109,26 @@ class HtmlAssembler(object):
         the configuration entry indra.config.get_config('INDRA_DB_REST_URL').
         If None, the URLs are constructed as relative links.
         Default: None
+    sort_by : str or function or None
+        If str, it indicates which parameter to sort by, such as 'belief' or
+        'ev_count', or 'ag_count'. Those are the default options because they
+        can be derived from a list of statements, however if you give a custom
+        stmt_sort_gather, you may use any of the parameters used to build it.
```

Comment:
Presumably this should be updated to read "if you provide customized attributes via the `custom_stats` argument"? (i.e., because stmt_sort_gather is not an arg of the assembler anymore)

Reply:
Good catch, I thought I got all of those updated references but obviously one slipped through.

Reply:
Ok, I did make these fixes. I must have forgotten to push before I re-loaded my OS. Oh well.